src/monster.php

```diff
@@ -1040,9 +1042,9 @@
 			echo '<table width="100%"><tr><td width="75%">';
 			echo "<b>Opções:</b> <a href=\"monster.php?act=attack&id=" . ($bixo->id * $player->id) . '">Atacar outr' . $enemy->prepo . " " . $enemy->username . "</a> | ";
 			if ($heal > 0 && $player->gold > $cost) {
-				echo "<a href=\"javascript:void(0)\" onclick=\"javascript:LoadPage('heal.php', 'swap')\">Recuperar vida</a> <font size=\"1\">(" . number_format($cost) . " de ouro)</font> | ";
+				echo "<a href=\"#\" onclick=\"javascript:LoadPage('heal.php', 'swap')\">Recuperar vida</a> <font size=\"1\">(" . number_format($cost) . " de ouro)</font> | ";
 			} elseif ($heal > 0 && $player->gold > 0) {
-				echo "<a href=\"javascript:void(0)\" onclick=\"javascript:LoadPage('heal.php', 'swap')\">Recuperar vida</a> <font size=\"1\">(" . number_format($cost2) . " de ouro)</font> | ";
+				echo "<a href=\"#\" onclick=\"javascript:LoadPage('heal.php', 'swap')\">Recuperar vida</a> <font size=\"1\">(" . number_format($cost2) . " de ouro)</font> | ";
```

Comment:
By changing the href from `javascript:void(0)` to `#` the page will be scrolled to the top, whereas not changing it would make the page not scroll, is it the intended behavior?